### PR TITLE
Added build_tags attribute support

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

File: tests/test_iib_client.py [hunk 1]
```diff
@@ -269,6 +271,7 @@ def test_iib_client(
                 organization="org",
                 overwrite_from_index=True,
                 overwrite_from_index_token="str",
+                build_tags=["extra-tag1"],
```
Comment (neithere) [inline tests/test_iib_client.py:274]: Shouldn't this be reflected in the result? We are comparing it to a model instance based on the fixture where `build_tags` is `["v4.5-2020-10-10"]`.

File: tests/test_iib_build_details_model.py [hunk 2]
```diff
@@ -51,12 +52,14 @@ def fixture_rm_build_details_json():
...
         "batch": 1,
         "batch_annotations": {"batch_annotations": 1},
+        "build_tags": ["v4.5-2020-10-10"],
         "logs": {},
         "deprecation_list": [],
         "updated": "updated",
         "user": "user@example.com",
         "binary_image": "binary_image",
         "binary_image_resolved": "binary_image_resolved",
+        "build_tags": [],
```
Comment (chandwanitulsi) [inline tests/test_iib_build_details_model.py:62]: Should we have 2 build_tags in this json?

File: iiblib/iib_build_details_model.py [hunk 3]
```diff
@@ -221,22 +221,24 @@ class AddModel(IIBBuildDetailsModel):
     __slots__ = [
         "binary_image",
         "binary_image_resolved",
-        "bundles",
```
Comment (querti) [inline iiblib/iib_build_details_model.py:224]: Should this be removed?

**Review by querti — CHANGES_REQUESTED** [review]

File: iiblib/iib_build_details_model.py (hunk 3 above)
Comment (midnightercz) [reply iiblib/iib_build_details_model.py:224]: it's no removed, just relocated to 226

**Review by querti — APPROVED** [review]

**Pull request merged**